openshift.yaml

```diff
@@ -0,0 +1,196 @@
+apiVersion: v1
```

Comment:
is it possible to move this in a subdirectory, like etc/openshift/ ?

Reply:
yes this file can be anywhere you like; I'll move it to `etc/openshift/` as suggested.